packages/backend/package.json

```diff
@@ -26,8 +26,8 @@
     "@koa/cors": "^4.0.0",
     "@koa/router": "^12.0.0",
     "@l2beat/backend-tools": "^0.4.0",
-    "@l2beat/discovery": "^0.8.0",
     "@l2beat/uif": "^0.2.2",
+    "@l2beat/discovery": "0.14.2",
```

Comment:
What happened to the order here? 😅

Reply:
`git rebase main` and applying both changes :trollface: